runtime-modules/proposals/codex/src/weights.rs

```diff
@@ -1019,6 +1020,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(7_u64))
 			.saturating_add(T::DbWeight::get().writes(9_u64))
 	}
+
+	fn create_proposal_decrease_council_budget(t: u32, d: u32, ) -> Weight {
+		Weight::from_parts(0, 0)
```

Comment:
This needs to be adjusted right?

Reply:
Hi, yes but they will be generated by @mnaamani on the ref. hardware

Reply:
And not on this pr